src/client/Microsoft.Identity.Client/Platforms/iOS/IntuneEnrollmentIdHelper.cs

```diff
@@ -52,6 +53,16 @@ internal static string GetRawEnrollmentId()
 #endif
         }
 
+        internal static string GetRawMAMResources()
+        {
+#if iOS
+            var keychainData = NSUserDefaults.StandardUserDefaults.StringForKey(Intune_MAMResourceKey);
```

Comment:
Does IntuneSDK put this resource into the KeyChain? What does it look like, is it just normal resource URI that we're trying to access?

Reply:
> Does IntuneSDK put this resource into the KeyChain? 
Yes
> What does it look like?
A json string.